### PR TITLE
97 docathon2 user guide

### LINKED ISSUE #97: Docathon2 - user guide - Coding Standards
Ok - so following review I am going to have stab at the "Coding Standards" section.

### PR DESCRIPTION
**Pull request by JhanSrbinovsky**
# CABLE

## Description

Please include a brief summary of the change and list the issues that are fixed.
Please also include relevant motivation and context.

You can link issues by using a supported keyword in the pull request's description or in a commit message:

Fixes #97 

## Documentation

- [ X] New or updated documentation

## Checklist

- [X ] The new content is accessible and located in the appropriate section.
- [ X] I have checked that links are valid and point to the intended content.
- [ X] I have checked my code/text and corrected any misspellings

## Comment

I was hoping I could link to the rendered code for an example of what the rendered MD will look like - but couldn't easily find it - so I gave up :)

### CODE REVIEW

**Comment by tammasloughran** [comment]
> I cant find the broken markdown link??

You can see by clicking on Details next to the checks.
```
=========================> MARKDOWN LINK CHECK <=========================

FILE: ./documentation/docs/POP-TRENDY/index.md
No hyperlinks found!

0 links checked.

FILE: ./documentation/docs/user_guide/uber_quick_guide.md
[✖] http://sites.fluxdata.org/AU-Tum/
[✓] index.md

2 links checked.

ERROR: 1 dead links found!
[✖] http://sites.fluxdata.org/AU-Tum/ → Status: 0

```

The failing link is the fluxnet data. The main branch has been updated to point to the fluxnet.org infor page for Tumbarumba instead.

I suggest merging the main branch into this branch first to get the new link.

**Comment by tammasloughran** [comment]
I think merging the main branch should fix the preview deployment also.

**Comment by JhanSrbinovsky** [comment]
My interpretation of it was that a link in my “code” was broken? But I checked the links in the VisualCode rendered version that I have locally and they appear fine. That’s what I mean about I cant find what link it is referring to – that I can fix.

From: Tammas Loughran ***@***.***>
Date: Thursday, 25 May 2023 at 11:38 am
To: CABLE-LSM/CABLE ***@***.***>
Cc: Srbinovsky, Jhan (Environment, Aspendale) ***@***.***>, Author ***@***.***>
Subject: Re: [CABLE-LSM/CABLE] 97 docathon2 user guide (PR #127)

I think merging the main branch should fix the preview deployment also.

File: documentation/docs/developer_guide/coding_standards.md
Comment (tammasloughran) [inline documentation/docs/developer_guide/coding_standards.md:1]: What are the general rules?

**Review by tammasloughran — CHANGES_REQUESTED** [review]
Section Coding standards seems to be missing a list of some sort.

The rest of the changes are text edits.

**Comment by ccarouge** [comment]
@JhanSrbinovsky definitely merge in `main` branch and you should be fine.

**Comment by JhanSrbinovsky** [comment]
Moved to ....rebranch #128

**Pull request closed without merging**